include/sound/sof/ipc4/header.h

```diff
@@ -192,6 +192,35 @@ enum sof_ipc4_pipeline_state {
 #define SOF_IPC4_GLB_LOAD_LIBRARY_LIB_ID_SHIFT	16
 #define SOF_IPC4_GLB_LOAD_LIBRARY_LIB_ID(x)	((x) << SOF_IPC4_GLB_LOAD_LIBRARY_LIB_ID_SHIFT)
 
+/* chain dma ipc message */
```

Comment:
suggested edit for the commit message

In the chained DMA mode, the firmware allocates buffers for the host and link DMA, and takes care of copying data between host- and link-DMA buffers in a low-latency thread. This is different to a regular pipeline, no processing is allowed, and the connection between host- and link DMA is handled with a dedicated IPC.

This patch exposes the macros needed to create the required IPC messages.

Reply:
Thanks! Writing a good description for a commit is often more difficult than writing the code for it. Especially when one do not yet know for certain all the relevant details 😅.